moduleframework/mtf.yaml

```diff
@@ -0,0 +1,61 @@
+default_module: docker
```

Comment:
this file will require a lot of documentation so users would understand what each key means

Reply:
thanks for these comments.
Yep, I expect that package will provide some default configuration (for testing upstream/fedora)
and some users who understand that, is able to reconfigure it to another value. 

Yep, would be nice to add there documentation. for now probably it is better without documentation, than have these values hardcoded inside python code.